packages/solecs/package.json

```diff
@@ -21,32 +21,23 @@
     "release": "npm publish --access=public",
     "test": "forge test"
   },
-  "devDependencies": {
+  "dependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
-    "@solidstate/contracts": "^0.0.52",
-    "@typechain/ethers-v5": "^10.2.0",
-    "@types/mocha": "^9.1.1",
     "ds-test": "https://github.com/dapphub/ds-test.git#c9ce3f25bde29fc5eb9901842bf02850dfd2d084",
     "ethers": "^5.7.2",
+    "memmove": "https://github.com/dk1a/memmove.git#ffd71cd77b1708574ef46a667b23ca3a5cc9fa27",
+    "solmate": "https://github.com/transmissions11/solmate.git#9cf1428245074e39090dceacb0c28b1f684f584c"
+  },
+  "devDependencies": {
+    "@solidstate/contracts": "^0.0.52",
+    "@typechain/ethers-v5": "^10.2.0",
     "forge-std": "https://github.com/foundry-rs/forge-std.git#b4f121555729b3afb3c5ffccb62ff4b6e2818fd3",
     "hardhat": "^2.10.2",
-    "memmove": "https://github.com/dk1a/memmove.git#ffd71cd77b1708574ef46a667b23ca3a5cc9fa27",
-    "rimraf": "^3.0.2",
     "solhint": "^3.3.7",
     "solidity-docgen": "^0.6.0-beta.22",
-    "solmate": "https://github.com/transmissions11/solmate.git#9cf1428245074e39090dceacb0c28b1f684f584c",
     "typechain": "^8.1.1",
     "typedoc": "0.23.21"
   },
-  "peerDependencies": {
-    "@ethersproject/abi": "^5.7.0",
-    "@ethersproject/providers": "^5.7.2",
-    "@solidstate/contracts": "^0.0.52",
-    "ds-test": "https://github.com/dapphub/ds-test.git#c9ce3f25bde29fc5eb9901842bf02850dfd2d084",
-    "ethers": "^5.7.2",
-    "forge-std": "https://github.com/foundry-rs/forge-std.git#b4f121555729b3afb3c5ffccb62ff4b6e2818fd3",
```

Comment:
moving `forge-std` to dependencies so it's installed automatically by consumers

Reply:
just a dev dep should be fine for forge-std